dhcpv4/client.go

```diff
@@ -212,6 +218,18 @@ func (c *Client) Exchange(ifname string, discover *DHCPv4, modifiers ...Modifier
 		return conversation, err
 	}
 
+	defer func() {
+		// close the sockets
+		if err := unix.Close(sfd); err != nil {
+			log.Printf("unix.Close(sendFd) failed: %v", err)
+		}
+		if sfd != rfd {
```

Comment:
Is sfd ever going to be the same as rfd with the current implementation?

Reply:
no, but since they are created separately, I want the code to be defensive to further changes